compiler/rustc_resolve/src/late.rs

```diff
@@ -1319,7 +1319,8 @@ impl<'a: 'ast, 'b, 'ast> LateResolutionVisitor<'a, 'b, 'ast> {
                 | PathSource::Struct
                 | PathSource::TupleStruct(..) => false,
             };
-            let mut error = false;
+            let mut error = true;
```

Comment:
`error` is redundant here, no?
It's always equivalent to `res == LifetimeRes::Error`.